user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Logging/Dashboards_and_Low_Code_Apps_Logging.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: Dashboards_and_Low_Code_Apps_Logging
+---
+
+# Logging
+
+The logging for Dashboards and Low Code Apps is located in `C:\Skyline DataMiner\Logging\Web`. There are two distinct types of logging, each serving specific purposes:
+
+- WebAPI logging
+
+- Client metric logging
+
+## WebAPI logging
+
+This category of logging records various events occurring within the WebAPI. Depending on the configuration, different types of logs can be enabled or disabled:
+
+- Debug logging provides information about various actions taken when handling a request from a client. It captures details about connections, unexpected errors, and so forth.
+
+- Allowed operation logging is utilized to record information about requests permitted or blocked by the Web Application Firewall (WAF) of a shared dashboard. It offers insights into unexpectedly blocked calls. It also logs information about WAF rules that take a long time to execute.
+
+## Client metric logging
+
+Client metric logging is employed to record different performance and issue indicators of clients connecting to Dashboards and Low Code Apps. These logs encompass everything from unexpected errors to the load time of a dashboard or LCA page. This logging data is stored in the `client` subfolder of the web logs.
+
+> [!NOTE]
+> This type of logging is available starting from DataMiner X/X onwards.
+
+## Configuration
+
+Both types of logging can be configured through a shared configuration file named `log.conf`, located in the root folder of the web logs. This file comprises various customizable settings:
+
+```md
```

Comment:
Why has this section been included as an md code block? Is this what the user will see in the file itself?
If you just want to show the table as part of the markdown file, it shouldn't be included in a code block.